utils/submissionDataFromGoogleMaps.ts

```diff
@@ -0,0 +1,196 @@
+import puppeteer from 'puppeteer'
+import axios from 'axios'
+import dotenv from 'dotenv'
+
+//Load the api key for google maps
+dotenv.config()
+const apiKey = process.env.GOOGLE_API_KEY
+
+// This uses all the helper functions in order to get the data we want
+export const getFacilityDetailsForSubmission = async (submittedURL: string): Promise<any | null> => {
```

Comment:
This is personal opinion, but I see utils as functionality that's independent of our app logic. (like logging or helper functions) 
Services hold the logic that's specific to our logic. (for example, you wouldn't copy to another project because it doesn't have "facilities") 

I think this could be split into a service and util. The specific code for submissions/domain logic should be in the service, and the generic/reusable google maps logic can be a `googleMapsUtil.ts`